src/js/formats/expandable.js

```diff
@@ -4,20 +4,33 @@ import expandableTemplate from '../text/expandable.dot.html!text';
 export default {
     preprocess({ content1: content, headline1: header, survey_like, survey_dislike }) {
         const words = content.split(' ');
-        const mainContentLength = 65;
-        const mainContent = words.slice(0, mainContentLength).join(' ');
-        const extraContent = words.slice(mainContentLength).join(' ');
+        const shortContentLength = 65;
+        const shortContent = words.slice(0, shortContentLength).join(' ');
 
-        return { header, mainContent, extraContent, survey_like, survey_dislike };
+        return {
+            header,
+            shortContent,
+            survey_like,
+            survey_dislike,
+            allContent: content,
+        };
     },
     postRender() {
-        q('.js-expand').forEach(el => el.addEventListener('click', ev => {
-            const readMoreLink = ev.currentTarget;
-
-            q('.js-extra-content').forEach(extraContentElement => {
-                extraContentElement.classList.remove('u-hidden');
+        q('.js-expand').forEach(el => el.addEventListener('click', () => {
```

Comment:
maybe we've got the point where we should use something like this: http://minifiedjs.com/